From bbc4245ce26df97100408961f33d57de08d2e063 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 28 Aug 2012 16:28:25 -0500
Subject: [PATCH 2006/2463] ENGR00215870-1 board-mx6 sabreauto rev B add extra
 pads

* For rev_b target board add extra pads table,
  separate pad definitions from I2C3 pad array

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.h    |    8 ++++----
 arch/arm/mach-mx6/board-mx6solo_sabreauto.h |    7 ++++---
 2 files changed, 8 insertions(+), 7 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.h b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
index 477078c..c087de6 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
@@ -282,16 +282,16 @@ static iomux_v3_cfg_t mx6q_i2c3_pads_rev_a[] __initdata = {
 static iomux_v3_cfg_t mx6q_i2c3_pads_rev_b[] __initdata = {
 	MX6Q_PAD_GPIO_3__I2C3_SCL,
 	MX6Q_PAD_EIM_D18__I2C3_SDA,
-	MX6Q_PAD_EIM_A24__GPIO_5_4,
-	MX6Q_PAD_GPIO_16__ENET_ANATOP_ETHERNET_REF_OUT,
 };
 static iomux_v3_cfg_t mx6q_tuner_pads[] __initdata = {
 	MX6Q_PAD_DISP0_DAT16__AUDMUX_AUD5_TXC,
 	MX6Q_PAD_DISP0_DAT18__AUDMUX_AUD5_TXFS,
 	MX6Q_PAD_DISP0_DAT19__AUDMUX_AUD5_RXD,
-
 };
-
+static iomux_v3_cfg_t mx6q_extra_pads_rev_b[] __initdata = {
+	MX6Q_PAD_EIM_A24__GPIO_5_4,
+	MX6Q_PAD_GPIO_16__ENET_ANATOP_ETHERNET_REF_OUT,
+};
 static iomux_v3_cfg_t mx6q_spinor_pads[] __initdata = {
 	/* eCSPI1 */
 	MX6Q_PAD_EIM_D16__ECSPI1_SCLK,
diff --git a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
index 2e9e495..9111e6b 100644
--- a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
@@ -283,15 +283,16 @@ static iomux_v3_cfg_t mx6dl_i2c3_pads_rev_a[] __initdata = {
 static iomux_v3_cfg_t mx6dl_i2c3_pads_rev_b[] __initdata = {
 	MX6DL_PAD_GPIO_3__I2C3_SCL,
 	MX6DL_PAD_EIM_D18__I2C3_SDA,
-	MX6DL_PAD_EIM_A24__GPIO_5_4,
-	MX6DL_PAD_GPIO_16__ENET_ANATOP_ETHERNET_REF_OUT,
 };
 static iomux_v3_cfg_t mx6dl_tuner_pads[] __initdata = {
 	MX6DL_PAD_DISP0_DAT16__AUDMUX_AUD5_TXC,
 	MX6DL_PAD_DISP0_DAT18__AUDMUX_AUD5_TXFS,
 	MX6DL_PAD_DISP0_DAT19__AUDMUX_AUD5_RXD,
 };
-
+static iomux_v3_cfg_t mx6dl_extra_pads_rev_b[] __initdata = {
+	MX6DL_PAD_EIM_A24__GPIO_5_4,
+	MX6DL_PAD_GPIO_16__ENET_ANATOP_ETHERNET_REF_OUT,
+};
 static iomux_v3_cfg_t mx6dl_spinor_pads[] __initdata = {
 	/* eCSPI1 */
 	MX6DL_PAD_EIM_D16__ECSPI1_SCLK,
-- 
1.7.7.4

